docs/competition/ros_communication.rst

```diff
@@ -48,6 +48,9 @@ Competition Topics
    * - :topic:`/ariac_human/state`
      - :term:`ariac_msgs/msg/HumanState`
      - Position and velocity of the human and the ceiling robot
+   * - :topic:`/ariac/assembly_insert_{n}_assembly_state`
+     - :term:`ariac_msgs/msg/AssemblyState`
```

Comment:
This message needs to be added to the glossary of messages as well.

Reply:
I have updated the message glossary. Please review.